Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp

```diff
@@ -744,8 +762,8 @@ namespace AzToolsFramework
 
     PropertyAssetCtrl::~PropertyAssetCtrl()
     {
+        AssetEditor::AssetEditorNotificationsBus::Handler::BusDisconnect();
         AssetSystemBus::Handler::BusDisconnect();
-        AzFramework::AssetCatalogEventBus::Handler::BusDisconnect();
```

Comment:
Seems like we should still be disconnecting from the `AssetCatalogEventBus` here, unless I am missing something.